pkg/simple/client/auditing/elasticsearch/options.go

```diff
@@ -22,6 +22,7 @@ import (
 )
 
 type Options struct {
+	Enabled     bool   `json:"enabled" yaml:"enabled"`
```

Comment:
I think `enable` is a better option here, `enabled` means component already enabled, it's a state, `enable` means to enable or not